frontend/src/components/UserTables/RidersContext.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Rider } from '../../types';
+import { useReq } from '../../context/req';
+
+type ridersState = {
+    getExistingRiders: () => Array<Rider>, 
+    refreshRiders: () => Promise<Array<Rider>> 
```

Comment:
getExistingRiders doesn't necessarily need to be a function, it can just be the `riders` variable that's created from `useState`

also refreshRiders can just have a return type of void, since it'll just update the value of `riders`